src/scss/global.scss

```
@@ -759,3 +759,28 @@ span.link {
 .flex-one {
   flex: 1;
 }
+
+// font size
+.font-size--small {
+  font-size: 0.9rem;
+}
```

Makes it cleaner/easier to add other variants in future:

```suggestion
.font-size {
  &--small {
    font-size: 0.9rem;
  }
}
```

---

fixed: https://github.com/sparkletown/sparkle/pull/1328/commits/e21e67445665543be0fab0f660743356141c30c7